src/web/components/App.tsx

```diff
@@ -28,6 +28,10 @@ import { getCommentContext } from '@root/src/web/lib/getCommentContext';
 
 type Props = { CAPI: CAPIBrowserType; NAV: NavType };
 
+const CommentsLayout = loadable(() =>
+    import('@frontend/web/components/CommentsLayout'),
+);
+
```

Comment:
Is it really the case that we need to refactor everything to use default exports? 😐

Reply:
I am scared this may be the only way, I will try dig deeper to see if we can find a work around. Maybe we can do `export default { CommentsLayout }` ? still figuring out what is best

Reply:
😬

Reply:
Relevant PR, no idea if it's expected soon... https://github.com/gregberge/loadable-components/pull/483

Reply:
Reading through that PR above, It seems like that's ready to ship - imminently hopefully. I've asked if that could happen so 🤞 

The API would be:

```js
const CommentsLayout = loadable(() => import('@frontend/web/components/CommentsLayout'), {
  resolveComponent: ({ CommentsLayout }) => () => <CommentsLayout />
});
```

which I think will be much cleaner.

I think it might be worth implementing this for discussion-rendering if that doesn't ship, but otherwise waiting for that new API.

Reply:
great find @gtrufitt yeah it would be amazing if they could release this